Doc/library/os.rst

```diff
@@ -1659,10 +1659,11 @@ or `the MSDN <https://msdn.microsoft.com/en-us/library/z0kc8e3z.aspx>`_ on Windo
       Added support for pipes on Windows.
 
 
-.. function:: splice(src, dst, count, offset_src=None, offset_dst=None)
+.. function:: splice(src, dst, count, offset_src=None, offset_dst=None, flags=0)
```

Comment:
Is flags pos-or-keyword or keyword-only?

Reply:
I think pos-or-keyword.